src/Services/WorksWithDatabases.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace Modelizer\Selenium\Services;
+
+trait WorksWithDatabases
+{
+    /**
+     * Assert that a given where condition exists in the database.
+     *
+     * @param string $table
+     * @param array  $data
+     * @param string $connection
+     *
+     * @return $this
+     */
+    protected function seeInDatabase($table, array $data, $connection = null)
+    {
+        if (is_null($this->app)) {
+            return; // don't run when there is no application
```

Comment:
Right now it doesn't matter, as in the overall setUp method we are explicitly declaring that we setup Laravel.  So the test will fail automatically if this package isn't used with Laravel.

These functions are also pulled from Laravel's Application Trait, I think it'd be fine to leave these for now, as I envision that we add a way to have this package work outside of Laravel itself, but if you would like the removed right now I can remove the checks and the test will fail anyway.